exercises/concept/elyses-analytic-enchantments/.docs/instructions.md

```diff
@@ -0,0 +1,66 @@
+# Instructions
+
+Elyse, magician-to-be, continues her training.
+She will be given several stacks of cards that she needs to perform her tricks.
```

Comment:
```suggestion
She will be given several stacks of cards that she needs to perform her tricks.
```

Maybe we should call these "decks" throughout to avoid confusing them with the "stack" datastructure.

Reply:
Good idea, I'll do that in a follow-up PR since it'd also affect the first elyse/vector exercise. Keeps the commit history slightly nicer.